package.json

```diff
@@ -168,12 +172,17 @@
     "temp": "~0.8.3",
     "update-notifier": "^2.5.0",
     "yargs": "^12.0.2",
+    "varint": "^5.0.0",
     "yargs-promise": "^1.1.0"
   },
   "optionalDependencies": {
     "prom-client": "^11.1.3",
     "prometheus-gc-stats": "~0.6.0"
   },
+  "peerDependencies": {
+    "electron-webrtc": "~0.3.0",
+    "wrtc": "~0.2.1"
+  },
```

Comment:
We get the following warning now when installing dependencies for js-ipfs:

```console
npm WARN ipfs@0.33.1 requires a peer of electron-webrtc@~0.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN ipfs@0.33.1 requires a peer of wrtc@~0.2.1 but none is installed. You must install peer dependencies yourself.
```

This makes people think that those dependencies are required when they are not. @achingbrain @hugomrdias do you think these should be moved to `optionalDependencies` instead?

Reply:
🙄 it shouldn't break...reading the docs, that seems to be the whole point of `optionalDependencies`...

Reply:
Ok, remove them for now as they were before...?